Pal/src/host/Linux-SGX/db_exception.c

```diff
@@ -62,6 +65,8 @@ void _DkGenericEventTrigger (PAL_IDX event_num, PAL_EVENT_HANDLER upcall,
     event.context = context;
     event.frame = frame;
 
+    if (rpc_queue)
```

Comment:
One thing we will need to be careful about is, anything imported from the URTS as an extern should probably be heavily commented on why this use of an untrusted input is safe (or, what is the worst thing that will happen if someone is adversarial).

Reply:
Indeed. I recognized now that `rpc_queue` needs some refactoring for security. I will add security checks on `rpc_queue` during enclave start-up, as well as comments.